src/FsAutoComplete/CodeFixes/AddPrivateAccessModifier.fs

```diff
@@ -111,7 +111,8 @@ let private getRangesAndPlacement input pos =
             members
             |> List.tryPick (fun m ->
               match m with
-              | SynMemberDefn.AutoProperty(accessibility = None; ident = ident; trivia = trivia) as a when
+              // TODO: Need to handle setting get or set accessibility separately
```

Comment:
Making a comment here to make a link to this todo for an issue - we need to be able to handle the other AST cases when the getter or setter have different visibilities.